### PR TITLE
Update eslint configs

### PR DESCRIPTION
**Pull request by mhagel**
Related: #19 

- remove amaury/eslintrc
- copy eslint config from regen-web
- use latest configs from regen-web dependency upgrades

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,9 +19,16 @@
...
+		"eslint-config-prettier": "^8.3.0",
+		"eslint-plugin-prettier": "^4.0.0",
 		"lerna": "^3.22.1",
+		"prettier": "2.4.1",
 		"ts-jest": "^26.4.4",
 		"typescript": "^4.0.5"
+	},
+	"dependencies": {
+		"@typescript-eslint/eslint-plugin": "^4.5.0",
+		"@typescript-eslint/parser": "^4.5.0"
```
Comment (blushi) [inline package.json:32]: these should probably be part of dev dep too

**Review by blushi — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (mhagel) [reply package.json:30]: done

**Review by blushi — APPROVED** [review]

File: .eslintrc.js [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+	],
+	rules: {
+		'no-empty': 'off',
+		'no-console': 'warn',
+		'no-useless-escape': 'off',
+		'@typescript-eslint/explicit-function-return-type': [
+		'error',
+			{
+				allowExpressions: true,
+			},
```
Comment (blushi) [inline .eslintrc.js:14]:
```suggestion
		{
				allowExpressions: true,
		},
```

Comment (mhagel) [reply .eslintrc.js:14]: fixed

**Review by haveanicedavid — APPROVED** [review]

**Pull request merged**